control-spec.txt

```diff
@@ -1537,14 +1537,17 @@
     "HSFETCH" SP (HSAddress / "v" Version "-" DescId)
               *[SP "SERVER=" Server] CRLF
 
-    HSAddress = 16*Base32Character
+    HSAddress = 16*Base32Character (v2) or 56*Base32Character (v3)
     Version = "2"
     DescId = 32*Base32Character
     Server = LongName
 
   This command launches hidden service descriptor fetch(es) for the given
   HSAddress or DescId.
 
+  HSAddress can be version 2 or version 3 addresses, but DescId can only be
+  version 2 IDs.
+
```

Comment:
We also have to mention that `DescID` is only for `Version=2`. And then either we mention that `SERVER=` is not supported in v3 or once we made the patch, we would be fine.